hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/controller/EidasCountrySelectedStateController.java

```diff
@@ -214,4 +214,23 @@ private State createEidasAuthnFailedErrorState() {
                 state.getCountryEntityId(),
                 state.getLevelsOfAssurance());
     }
+
+    @Override
+    public void transitionToSessionStartedState() {
+        final SessionStartedState sessionStartedState = createSessionStartedState();
+        hubEventLogger.logSessionMovedToStartStateEvent(sessionStartedState);
+        stateTransitionAction.transitionTo(sessionStartedState);
+    }
+
+    private SessionStartedState createSessionStartedState() {
+        return new SessionStartedState(
+                state.getRequestId(),
+                state.getRelayState().orNull(),
+                state.getRequestIssuerEntityId(),
+                state.getAssertionConsumerServiceUri(),
+                null,
```

Comment:
Why is this `null`?

Reply:
Turns out this is actually an issue – we need to be preserving the state of the `ForceAuthentication` flag so that if the user switches from Verify to eIDAS and back again we still pass the flag to the IDP.

Reply:
That's correct - I will raise a separate PR for that

Reply:
Resolved in https://github.com/alphagov/verify-hub/pull/161